tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -4412,4 +4412,58 @@ public function test_add_email_currency_conversion_notice_outputs_plain_text_for
+			->disableOriginalConstructor()
+			->onlyMethods( [ 'empty_cart' ] )
+			->getMock();
+
+		$mock_cart->expects( $this->once() )->method( 'empty_cart' );
+
+		WC()->cart = $mock_cart;
+
+		$response = $this->mock_gateway->process_payment( $order_id );
+
+		$this->assertEquals( 'success', $response['result'] );
+	}
```

PHPMD (2.15.0)

[error] test_process_payment_with_payment_method_clears_cart_on_success accesses the super-global variable $_POST. (undefined)

(Superglobals)